src/main.tf

```
@@ -16,7 +16,7 @@ locals {
 
   authorization_rules = [
     for private_subnet_cidr in module.vpc.outputs.private_subnet_cidrs : {
-      name                 = "Internal Rule"
+      name                 = "Internal Rule - ${private_subnet_cidr}"
```

I saw this as well, and thanks for submitting the fix! Can we tweak this however? This name is used in the Terraform resource name as the key, so it's better to avoid spaces and capitalization

```suggestion
      name                 = "internal.${private_subnet_cidr}"
```